v0/template.json

```diff
@@ -66167,6 +66167,15 @@
               "key" : {
                 "type" : "string"
               },
+              "override" : {
```

Comment:
Can we please check with the product team on what this should be called? Maybe `force_push` is more readable?